js/data.js

```diff
@@ -0,0 +1,87 @@
+import {getRandomArrayElement} from './util.js';
```

Comment:
Один файл - одно объявление импорта

```suggestion
import {
  getRandomArrayElement,
  getRandomPositiveInteger,
  createRandomIdGenerator,
} from './util.js';
```